### ISSUE TITLE
chore: add `Slovenian - sl-SI` language support

### ISSUE THREAD

**Issue by brionmario** [asgardeo/javascript#200]
### Current Limitation

Currently, the Asgardeo JavaScript SDK does not have support for **Slovenian (Slovenščina - sl-SI)**. Users who prefer Slovenian will not see UI components translated, limiting accessibility and localization for Slovenian-speaking users in Slovenia and globally.

> [!Important]
> Contributors for this task should be **native Slovenian speakers or have professional fluency** to ensure accurate, culturally appropriate translations.

### Suggested Improvement

Add **Slovenian (`sl-SI`) language support** to the `@asgardeo/i18n` package.

**Implementation Plan:**

1. Create a new locale file:

    Follow https://github.com/asgardeo/javascript/blob/main/CONTRIBUTING.md#contributing-to-the-internalization-i18n-package

2. Add documentation for usage:  
   - Update the SDK docs: https://wso2.com/asgardeo/docs/sdks/react/overview/

### Please select the package issue is related to

@asgardeo/i18n

### Version

current

### Reporter Checklist

- [x] I have searched the existing issues and this is not a duplicate.
- [x] I have provided all the necessary information.
- [x] I have verified the improvement is not available in the latest version of the package.

**Comment by MaticSulc** [comment]
Hey! As requested in PR, I'd like to get this issue assigned, since I'm a native speaker. Thanks!

**Comment by sagara-gunathunga** [comment]
@MaticSulc I've assigned this issue to you.
